appserver/admingui/devtests/run_test.sh

```diff
@@ -58,6 +58,10 @@ test_run(){
   chmod 600 $APS_HOME/password.txt
   echo "AS_ADMIN_PASSWORD=" > $APS_HOME/password.txt
   echo "AS_ADMIN_NEWPASSWORD=$PWD" >> $APS_HOME/password.txt
+  LOCKFILE=$S1AS_HOME/domains/domain1/imq/instances/imqbroker/lockv
```

Comment:
Can you please add a comment why this is needed?

Reply:
Deploy was failing due to MQJMSRA_RA4001: start:Aborting:Exception starting EMBEDDED broker=Cannot create broker instance.  A broker instance is already created.]].

This generally happens when this lock file is created as part of some back end code. In Jenkins, sometimes this file is getting created. I got to know this from backend team.
Hence, deleting the file before running the tests.